collectors/python.d.plugin/python.d.conf

```diff
@@ -62,6 +62,7 @@ logind: no
 # ntpd: yes
 # openldap: yes
 # oracledb: yes
+# pandas: no
```

Comment:
I don't see the collector being disabled.

Choose one:
 - `# pandas: yes`
 - `pandas: no` + `disabled_by_default = True` in the src

Reply:
i though i was being extra safe with `# pandas: no` instead of `# pandas: yes`. 

will change

Reply:
changed to `# pandas: yes`